cmd/kraft/kraft.go

```diff
@@ -137,6 +137,7 @@ func main() {
 	if !config.G[config.KraftKit](ctx).NoCheckUpdates {
 		if err := kitupdate.Check(ctx); err != nil {
 			log.G(ctx).Debugf("could not check for updates: %v", err)
+			log.G(ctx).Debug("if you would like to disable update checks, set `no_check_updates: true` in the configuration file")
```

Comment:
Let's make this closer to [the existing message](https://github.com/unikraft/kraftkit/blob/staging/internal/update/update.go#L68-L88) with something like:
```go
log.G(ctx).Debugf("could not check for updates: %v", err)
log.G(ctx).Debug("")
log.G(ctx).Debug("to turn off this check, set:")
log.G(ctx).Debug("")
log.G(ctx).Debug("\texport KRAFTKIT_NO_CHECK_UPDATES=true")
log.G(ctx).Debug("")
log.G(ctx).Debug("or use the globally accessible flag '--no-check-updates'")
```